api/v1beta1/verticadb_types.go

```diff
@@ -94,15 +94,15 @@ type VerticaDBSpec struct {
 	ShardCount int `json:"shardCount"`
 
 	// +kubebuilder:validation:Optional
-	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors={"urn:alm:descriptor:io.kubernetes:Secret"}
 	// An optional name for a secret that contains the password for the
 	// database's superuser. If this is not set, then we assume no such password
 	// is set for the database. If this is set, it is up the user to create this
 	// secret before deployment. The secret must have a key named password.
 	SuperuserPasswordSecret string `json:"superuserPasswordSecret,omitempty"`
 
 	// +kubebuilder:validation:Optional
-	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors={"urn:alm:descriptor:io.kubernetes:Secret"}
```

Comment:
Why is this one defined with a brace?  i.e. {"..."} instead of "...".

Reply:
Yeah, I will change that. Braces are for many not one.